pyproject.toml

```diff
@@ -18,23 +18,28 @@ dependencies = [
 ]
 name = "lando"
 requires-python = ">=3.10"
-version = "0.1.0"
+dynamic = ["version"]
 
 [project.scripts]
 lando = "lando.manage:main"
 
 [build-system]
 requires = [
-    "setuptools>=45",
+    "setuptools>=64",
+    "setuptools_scm>=8",
     "wheel",
 ]
 build-backend = "setuptools.build_meta"
 
+[tool.setuptools_scm]
+version_file = "src/lando/version.py"
+
 [tool.setuptools.packages.find]
 where = ["src"]
 
 [tool.pytest.ini_options]
 DJANGO_SETTINGS_MODULE = "lando.test_settings"
 testpaths = [
     "src/lando/api",
-]
+	"src/lando/tests",
```

Comment:
Indentation looks funky, looks like you need to run this through `black`.

Reply:
There's a `lando format` command by the way. There are some inconsistencies in terms of formatting across the repo, mainly due to the two lando legacy repos having different formatting. I filed a separate bug to track the work of fixing that (https://bugzilla.mozilla.org/show_bug.cgi?id=1897434), until then it's OK to manually run `ruff` and `black` (within the container) on files that are changed.

Reply:
It's because it's a tab, I think I opened this file in notepad to add the line (instead of my usual IDE) and it didn't replace the tab with four spaces.

I did actually run `lando format`, and just tried again, and it doesn't replace the tab with four spaces. I've manually corrected it though.

Reply:
I guess we both missed that this is not a Python file (and also, is in the root directory of the project.)

Reply:
Looks like `ruff` would have caught this, but it is not defined in LINT_PATHS. Can be added there.